context/app/static/js/components/HRA/CCFOrganInfo.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import reactifyWc from 'reactify-wc';
```

Comment:
Are we concerned that the last publish was 3 years ago?

https://www.npmjs.com/package/reactify-wc

Reply:
I'm open to changing this down the line; the more recent wrappers I found wanted the component's JS Class as well, which would require @bherr2's team to publish the components on `npm`.

- https://www.npmjs.com/package/@microsoft/fast-react-wrapper
  - API constructs a wrapper using the current React runtime then wraps it around the component's Class
- https://www.npmjs.com/package/@lit/react
  - API includes `elementClass`
- https://www.npmjs.com/package/wc-react
  - Only other wrapper I found where the custom component's tag is wrapped instead of the class, but it's similarly unmaintained (one year vs. three)

Hopefully by the time that the age of this dependency begins affecting us, React will have mainline support for web-components, as the `experimental` version currently does and we will be able to remove it altogether: https://custom-elements-everywhere.com/libraries/react-experimental/results/results.html